lib/wit/common.py

```diff
@@ -16,3 +17,11 @@ class WitUserError(Exception):
     Supertype of user-input errors that should be reported without stack traces
     """
     pass
+
+
+# https://stackoverflow.com/a/845194
+def passbyval(func):
+    def new(self, *args):
```

Comment:
You might consider implementing `**kwargs` as well since if anyone tries to call a function decorated with this using keyword arguments, it will just silently drop them

Reply:
fixed by https://github.com/sifive/wit/pull/96/commits/61d0d7b7dade20530295f3ecd26e9ad38446141c